### PR TITLE
Relics can be added using the give all command

### PR DESCRIPTION
**Pull request by cfuncode**
When using the give all command, added the maximum level of relic for the corresponding star.
However, there are many values after the secondary attribute upgrade that do not belong to the normal range.
And don't know how to fix it.

### CODE REVIEW

**Comment by ghost** [comment]
Related, feature previously removed by https://github.com/Grasscutters/Grasscutter/pull/365

File: src/main/java/emu/grasscutter/command/commands/ClearCommand.java [hunk 1]
```diff
@@ -42,7 +42,8 @@ public void execute(Player sender, Player targetPlayer, List<String> args) {
             case "art" -> {
             	toDelete = playerInventory.getItems().values().stream()
                         .filter(item -> item.getItemType() == ItemType.ITEM_RELIQUARY)
-                        .filter(item -> item.getLevel() == 1 && item.getExp() == 0)
```
Comment (ghost) [inline src/main/java/emu/grasscutter/command/commands/ClearCommand.java:45]: Note this changes logic of `clear`

Comment (cfuncode) [reply src/main/java/emu/grasscutter/command/commands/ClearCommand.java:45]: Isn't this the right change? Only sacred relics worn and locked will not be deleted. However, the Readme. md content in English has not been corrected.

**Comment by cfuncode** [comment]
> Related, feature previously removed by #365

It wasn't removed, was it? Isn't it still working?

**Pull request closed without merging**